pkg/cmd/cli/describe/describer.go

```diff
@@ -283,17 +283,16 @@ func (d *BuildConfigDescriber) Describe(namespace, name string) (string, error)
 		if len(builds.Items) == 0 {
 			return nil
 		}
-		fmt.Fprintf(out, "Builds(Name,Status,Creation Time)\n")
+		fmt.Fprintf(out, "Builds(Name,Status,Duration,Creation Time)\n")
 		sortedBuilds := sortableBuilds(builds.Items)
 		sort.Sort(sortedBuilds)
-		c := 0
 		for i := range sortedBuilds {
 			// iterate backwards so we're printing the newest items first
 			build := sortedBuilds[len(sortedBuilds)-1-i]
-			fmt.Fprintf(out, fmt.Sprintf("- %s %s %v\n", build.Name, build.Status, build.CreationTimestamp.Rfc3339Copy().Time))
-			c++
+			duration := build.CompletionTimestamp.Rfc3339Copy().Time.Sub(build.StartTimestamp.Rfc3339Copy().Time)
```

Comment:
Yeah, my idea would be to either:
1. add `Duration` field to `Build` object that will be filled upon get (I don't think we should store that)
2. replace `CompletionTimestamp` with `Duration`, IMHO these two make more sense than having start and end time.
Personally I'm leaning towards the latter, what do you think @bparees @smarterclayton ?